packages/airnode-node/src/api/index.ts

```diff
@@ -227,6 +229,11 @@ async function processSuccessfulApiCall(
         );
         return [[], { success: true, value: JSON.stringify({ timestamp, value: response.encodedValue }), signature }];
       }
+      case 'beacon': {
```

Comment:
What if we just use the `http-gateway` type? 🤔

Reply:
> What if we just use the http-gateway type?

Good point, I wasn't sure if it would be weird to have that on the Airkeeper (or if there would be need for more Beacon specific types in Airnode later on)

I can make the change to `http-gateway` if that's preferred

Reply:
I'm not sure tbh so I'd like @amarthadan opinion on this since he's more familiar with this code these days.

Reply:
Actually, I think we may be able to let **airnode-node** do the `templateId` verification so this could warrant a separate type :thinking:

Reply:
🤔 Legacy template Ids will include the [airnode address](https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/contracts/rrp/TemplateUtils.sol#L9) in the hash and [psp beacons won't](https://github.com/api3dao/airnode/blob/v1-proto-tob9-fix/packages/airnode-protocol-v1/contracts/protocol/StorageUtils.sol#L20). I'm not sure we can rely on the node's template verification function for all cases.

Reply:
> What is the plan here? Does airkeeper want to call airnode to perform the API call instead doing the API call directly?

Right, that's the end goal

Reply:
> Legacy template Ids will include the [airnode address](https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/contracts/rrp/TemplateUtils.sol#L9) in the hash and [psp beacons won't](https://github.com/api3dao/airnode/blob/v1-proto-tob9-fix/packages/airnode-protocol-v1/contracts/protocol/StorageUtils.sol#L20). I'm not sure we can rely on the node's template verification function for all cases.

Ah I see, I may have seem a comment about this somewhere  :smile:

Reply:
@Siegrift Just to be clear, Airkeeper won't call a running Airnode (like invoking a Lambda or anything). We just want to reuse the `callApi` code in Airkeeper. The request will still be actually done by Airkeeper.

@vponline @acenolaza I'd say let's reuse `http-gateway` type (we might consider renaming it in the future) for now, we can diverse later if need be. And let's keep the template verification for Airkeeper since that would require bigger refactoring on the Airnode side.